iam-persistence/src/main/java/it/infn/mw/iam/persistence/model/IamUserInfo.java

```diff
@@ -91,6 +91,9 @@ public class IamUserInfo implements Serializable {
 
   private String birthdate;
 
+  @Column(length = 4096)
```

Comment:
we're expecting this value as a University or Institute name so having 4096 characters for this value is, in my opinion, too much. I think 200 should be greater enough to fit what we need. What do you think?